### DISCUSSION TITLE
Welcome to VHS Discussions!

### DISCUSSION THREAD

**Discussion by maaslalani** [charmbracelet/vhs#40] [Announcements]
## VHS Discussions

Welcome to VHS discussions. Feel free to ask any questions or suggest any improvements for VHS!

Also, feel free to share any renders you've made using VHS!

**Comment by danon** [comment]
This project is soooo coool!

**Comment by igorcarneir0** [comment]
@maaslalani I'm trying to install VHS on my machine (Windows) but it's very complicated for me, I don't have much knowledge in the area yet. Do you have a step-by-step tutorial? ;(
